### ISSUE TITLE
[Suggestion] Update Team Members on About Page 

### ISSUE THREAD

**Issue by greensteph** [GSA/digitalgov.gov#2078]
### URL / Link to page
https://digital.gov/about/#our-team

### Tell us how we can make this page better
Outdated team members list: 
<hr>

![Screenshot 2020-03-02 07 42 27](https://user-images.githubusercontent.com/11343071/75686450-81569000-5c59-11ea-89bc-15971a205908.png)

**Comment by jeremyzilar** [comment]
Thank you @greensteph —— this is bundled up in https://github.com/GSA/digitalgov.gov/pull/2015 but happy to pull this out now into a separate PR and unblock it :)

**Comment by jeremyzilar** [comment]
🌈  https://github.com/GSA/digitalgov.gov/pull/2163
